src/promptflow-tools/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Release History
 
+## 1.5.0 (Upcoming)
+
+### Features Added
+- Add "detail" to "Azure OpenAI GPT-4 Turbo with Vision" and "OpenAI GPT-4V" tool inputs.
```

Comment:
Support "detail" in "Azure OpenAI GPT-4 Turbo with Vision" and "OpenAI GPT-4V" tool.

Reply:
Seems not much difference. I am following the sentence pattern we used for "seed" parameter in 1.4.0